src/__tests__/organizationProposals.int.test.ts

```diff
@@ -14,11 +14,11 @@ const agent = request.agent(app);
 
 const insertTestOrganizations = async () => {
 	await createOrganization({
-		employerIdentificationNumber: '11-1111111',
+		taxId: '11-1111111',
 		name: 'Example Inc.',
 	});
 	await createOrganization({
-		employerIdentificationNumber: '22-2222222',
```

Comment:
I don't believe any clients are creating organizations via API so I don't think we have to maintain both values for the create case, just the return values.